stable/elasticsearch-exporter/README.md

```diff
@@ -66,8 +66,10 @@ Parameter | Description | Default
 `podSecurityPolicies.enabled` | Enable/disable PodSecurityPolicy and associated Role/Rolebinding creation | `false`
 `serviceAccount.create` | Create a ServiceAccount for the pod | `false`
 `serviceAccount.name` | Name of a ServiceAccount to use that is not handled by this chart | `default`
-`service.type` | type of service to create | `ClusterIP`
+`service.type` | type of service to create, `ClusterIP` or `NodePort` | `ClusterIP`
```

Comment:
There are other valid types.

Reply:
should I and loadbalancer and ingress?

Reply:
I would just leave it as `type of service to create`

But FWIW take a look on other charts and see how they do...